frontend/src/suspense/AppLoader.jsx

```diff
@@ -0,0 +1,14 @@
+//Taken from https://www.copycat.dev/blog/react-suspense/
+import "./appLoader.css";
+
+export const AppLoader = () => {
+  return (
+    <section className="app-loader">
+      <div className="bouncing-loader">
+        <div></div>
+        <div></div>
+        <div></div>
```

Comment:
are these divs necessary?

Reply:
Yup, they contain the dots that bounce on the loading screen. I'm going to add a descriptive className